client/src/extension.ts

```diff
@@ -65,7 +65,7 @@ function updatePythonPath (): void {
   for (const pythonSubConf of ['autoComplete.extraPaths', 'analysis.extraPaths']) {
     const extraPaths = pythonConfig.get<string[]>(pythonSubConf) ?? []
     for (const pathToAdd of [pathToBitbakeLib, pathToPokyMetaLib]) {
-      if (!extraPaths.includes(pathToAdd)) {
+      if (!extraPaths.includes(pathToAdd) && Object.isExtensible(extraPaths)) {

```

Comment:
How did you run into that condition? Did you override the setting with a bugous value? Did you run the client inside another environment? Is it just a linter warning? the line above defines an array in any case:
`const extraPaths = pythonConfig.get<string[]>(pythonSubConf) ?? []`
Also In that case, it could make more sense to define a new array instead of ignoring the extraPaths. We still would want to create a new array with our Python paths, not discard them.

Reply:
> Also In that case, it could make more sense to define a new array instead of ignoring the extraPaths

Can you demo how to define a new array?

Reply:
```typescript
const extraPaths = pythonConfig.get<string[]>(pythonSubConf) ?? []
if(!Object.isExtensible(extraPaths)) extraPaths = []
for (const pathToAdd of [pathToBitbakeLib, pathToPokyMetaLib]) {
    if (!extraPaths.includes(pathToAdd)) {
```